### ISSUE TITLE
Create a twolame.spec

### ISSUE THREAD

**Issue by njh** [njh/twolame#10]

**Comment by njh** [comment]
Created .spec file with a 3-way split:
* twolame  (command line tool and man page)
* twolame-devel (header files and libraries for developing against)
* twolame-libs (shared libraries)


```
[developer@sandbox7 ~]$ rpm -ql twolame
/usr/bin/twolame
/usr/share/doc/twolame-0.4.0
/usr/share/doc/twolame-0.4.0/AUTHORS
/usr/share/doc/twolame-0.4.0/COPYING
/usr/share/doc/twolame-0.4.0/ChangeLog
/usr/share/doc/twolame-0.4.0/NEWS
/usr/share/doc/twolame-0.4.0/README
/usr/share/man/man1/twolame.1.gz
```

```
[developer@sandbox7 ~]$ rpm -ql twolame-devel
/usr/include/twolame.h
/usr/lib64/libtwolame.a
/usr/lib64/libtwolame.la
/usr/lib64/libtwolame.so
/usr/lib64/pkgconfig/twolame.pc
/usr/share/doc/twolame
/usr/share/doc/twolame-0.4.0
/usr/share/doc/twolame-0.4.0/AUTHORS
/usr/share/doc/twolame-0.4.0/COPYING
/usr/share/doc/twolame-0.4.0/ChangeLog
/usr/share/doc/twolame-0.4.0/NEWS
/usr/share/doc/twolame-0.4.0/README
/usr/share/doc/twolame/api.txt
/usr/share/doc/twolame/psycho.txt
/usr/share/doc/twolame/vbr.txt
```

```
[developer@sandbox7 ~]$ rpm -ql twolame-libs
/usr/lib64/libtwolame.so.0
/usr/lib64/libtwolame.so.0.0.0
/usr/share/doc/twolame-libs-0.4.0
/usr/share/doc/twolame-libs-0.4.0/COPYING
```

```
[developer@sandbox7 ~]$ rpm -ql twolame-debuginfo
/usr/lib/debug
/usr/lib/debug/.build-id
/usr/lib/debug/.build-id/2e
/usr/lib/debug/.build-id/2e/c2fe93eb63472f6c88b0b2ea22d69474143d82
/usr/lib/debug/.build-id/2e/c2fe93eb63472f6c88b0b2ea22d69474143d82.debug
/usr/lib/debug/.build-id/38
/usr/lib/debug/.build-id/38/b61a5421437d7aff7248934f2b7253526ebfb8.debug
/usr/lib/debug/.build-id/ed
/usr/lib/debug/.build-id/ed/16a5cabd5a3a2ed9bbb83c9fc2e20dfe1b1ce9
/usr/lib/debug/.build-id/ed/16a5cabd5a3a2ed9bbb83c9fc2e20dfe1b1ce9.debug
/usr/lib/debug/.dwz
/usr/lib/debug/.dwz/twolame-0.4.0-3.bbc.el7.x86_64
/usr/lib/debug/usr
/usr/lib/debug/usr/bin
/usr/lib/debug/usr/bin/twolame.debug
/usr/lib/debug/usr/lib64
/usr/lib/debug/usr/lib64/libtwolame.so.0.0.0.debug
/usr/lib/debug/usr/lib64/libtwolame.so.0.debug
/usr/lib/debug/usr/lib64/libtwolame.so.debug
/usr/src/debug/twolame-0.4.0
/usr/src/debug/twolame-0.4.0/frontend
/usr/src/debug/twolame-0.4.0/frontend/audioin_raw.c
/usr/src/debug/twolame-0.4.0/frontend/audioin_sndfile.c
/usr/src/debug/twolame-0.4.0/frontend/frontend.c
/usr/src/debug/twolame-0.4.0/frontend/frontend.h
/usr/src/debug/twolame-0.4.0/libtwolame
/usr/src/debug/twolame-0.4.0/libtwolame/ath.c
/usr/src/debug/twolame-0.4.0/libtwolame/ath.h
/usr/src/debug/twolame-0.4.0/libtwolame/availbits.c
/usr/src/debug/twolame-0.4.0/libtwolame/availbits.h
/usr/src/debug/twolame-0.4.0/libtwolame/bitbuffer.c
/usr/src/debug/twolame-0.4.0/libtwolame/bitbuffer.h
/usr/src/debug/twolame-0.4.0/libtwolame/bitbuffer_inline.h
/usr/src/debug/twolame-0.4.0/libtwolame/common.h
/usr/src/debug/twolame-0.4.0/libtwolame/crc.c
/usr/src/debug/twolame-0.4.0/libtwolame/crc.h
/usr/src/debug/twolame-0.4.0/libtwolame/dab.c
/usr/src/debug/twolame-0.4.0/libtwolame/dab.h
/usr/src/debug/twolame-0.4.0/libtwolame/encode.c
/usr/src/debug/twolame-0.4.0/libtwolame/encode.h
/usr/src/debug/twolame-0.4.0/libtwolame/energy.c
/usr/src/debug/twolame-0.4.0/libtwolame/energy.h
/usr/src/debug/twolame-0.4.0/libtwolame/enwindow.h
/usr/src/debug/twolame-0.4.0/libtwolame/fft.c
/usr/src/debug/twolame-0.4.0/libtwolame/fft.h
/usr/src/debug/twolame-0.4.0/libtwolame/get_set.c
/usr/src/debug/twolame-0.4.0/libtwolame/mem.c
/usr/src/debug/twolame-0.4.0/libtwolame/mem.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_0.c
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_0.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_1.c
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_1.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_1_critband.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_1_freqtable.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_2.c
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_2.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_2_absthr.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_3.c
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_3.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_4.c
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_4.h
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_n1.c
/usr/src/debug/twolame-0.4.0/libtwolame/psycho_n1.h
/usr/src/debug/twolame-0.4.0/libtwolame/subband.c
/usr/src/debug/twolame-0.4.0/libtwolame/subband.h
/usr/src/debug/twolame-0.4.0/libtwolame/twolame.c
/usr/src/debug/twolame-0.4.0/libtwolame/twolame.h
/usr/src/debug/twolame-0.4.0/libtwolame/util.c
/usr/src/debug/twolame-0.4.0/libtwolame/util.h
```

**Comment by njh** [comment]
twolame.spec added in d31d75aafdff0be9e241628338faef11bd13f4e6.

I need to check why some files are in /usr/share/doc/twolame-0.4.0/ and some in /usr/share/doc/twolame/

**Comment by njh** [comment]
I looked into this a bit:

* The `%doc` macro puts files into /usr/share/doc/twolame-0.4.0/
* But automake sets `pkgdocdir` / `@docdir@` to /usr/share/doc/twolame/ (DATAROOTDIR/doc/twolame)

The Fedora project plans to change to installing docs in /usr/share/doc/twolame/:
* https://fedoraproject.org/wiki/Changes/UnversionedDocdirs

This is being done by changing %_docdir_fmt from `%%{NAME}-%%{VERSION}` to `%%{NAME}`.
CentOS 7 doesn't seem to have received this change yet.

I am tempted to stop using the %doc macro and have automake install the `AUTHORS COPYING ChangeLog README NEWS` files instead.

**Comment by njh** [comment]
Resolved documentation install location in 4258d8e92a5637fa091186df54e706bc29336e7b
